README.md

```diff
@@ -29,7 +28,39 @@ $ codeclimate-test-reporter
 Submitting payload to https://codeclimate.com
 ```
 
-### Alternate .coverage file location
+### Generating Coverage Reports
+
+To generate a coverage report with [pytest][], you can use the [pytest-cov][]
+plugin:
+
+```console
+$ py.test --cov=codeclimate_test_reporter tests/
+TOTAL                                                     284     27    90%
+
+======================== 14 passed in 0.75 seconds ========================
+```
+
+To generate a coverage report with [nose][], you can use the [nose cover plugin][]:
+
+[pytest]: http://pytest.org/
+[nose]: https://nose.readthedocs.org
+[pytest-cov]: https://pypi.python.org/pypi/pytest-cov
+[nose cover plugin]: https://nose.readthedocs.org/en/latest/plugins/cover.html
+
+```console
+$ nosetests --with-coverage --cover-erase --cover-package=codeclimate_test_reporter
+TOTAL                                                284     27    90%
+----------------------------------------------------------------------
+Ran 14 tests in 0.743s
+
+OK
+```
+
+By default, coverage.py will create the test coverage report at `./.coverage`.
+Simply run the `codeclimate-test-reporter` to send your test report to
```

Comment:
I would actually rethink this whole sentence: this is several sections before the important information about configuration (with the repo token, etc.), so "simply running `codeclimate-test-reporter`" is unlikely to actually work.

We could probably drop this sentence completely, I think: the other headings point the user in the right direction. If anything, perhaps changes should be made to other sections: maybe a section like "Uploading your coverage report" would be more instructive than "CI Integration".